### PR TITLE
TC Automation- RestartPXWhileVolCreate 

### PR DESCRIPTION
**Pull request by spai-px**
What this PR does / why we need it:
This PR adds RestartPXWhileVolCreate test case which checks no error occurs when PX is restarted while creation and deletion of FADA nginx apps PVC 

Which issue(s) this PR fixes (optional)
Closes #PTX-20642

Special notes for your reviewer:
[Aetos dashboard](https://aetos.pwx.purestorage.com/resultSet/testSetID/393444)

### CODE REVIEW

File: tests/basic/pure_test.go [hunk 1]
```diff
@@ -335,3 +336,82 @@ var _ = Describe("{BringUpLargePodsVerifyNoPanic}", func() {
...
+
+// This test Creates multiple FADA volume/app (nginx) - Restart PX on a Node while volume creation is in progress
+/*
+https://portworx.testrail.net/index.php?/tests/view/72615026
+
+*/
+var _ = Describe("{RestartPXWhileVolCreate}", func() {
+	var contexts []*scheduler.Context
+	JustBeforeEach(func() {
+		StartTorpedoTest("RestartPXWhileVolCreate", "Test creates multiple FADA volume and restarts px on a node while volume creation is in progress", nil, 0)
```
Comment (kphalgun-px) [inline tests/basic/pure_test.go:348]: Please replace 0 with 72615026 (TestRailID).

Comment (spai-px) [reply tests/basic/pure_test.go]: Done

File: tests/basic/pure_test.go [hunk 2]
```diff
@@ -335,3 +337,108 @@ var _ = Describe("{BringUpLargePodsVerifyNoPanic}", func() {
...
+				}()
+
+				// Step 2: Restart Portworx
+				wg.Add(1)
+				go func() {
+					defer wg.Done()
+					stepLog := "Restart Portworx"
+					Step(stepLog, func() {
+						nodes := node.GetStorageDriverNodes()
+						randomNode := rand.Intn(len(nodes))
```
Comment (lsrinivas-pure) [inline tests/basic/pure_test.go:389]: how are you making sure, node where volume is getting created in restarted.
you might need to use volume placement strategies for this

Comment (kphalgun-px) [reply tests/basic/pure_test.go:389]: Can we inspect volume and do a restart based on response?

Comment (spai-px) [reply tests/basic/pure_test.go:389]: Picking of random nodes was suggested by hari. So I went ahead with it. Maybe we can discuss this further.

Comment (spai-px) [reply tests/basic/pure_test.go]: Done with placing volumes in the selected node and restarting px in that particular node.

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**